laws/src/main/scala/cats/laws/discipline/TraverseFilterTests.scala

```diff
@@ -58,8 +68,10 @@ trait TraverseFilterTests[F[_]] extends FunctorFilterTests[F] {
       "filterA consistent with traverseFilter" -> forAll(laws.filterAConsistentWithTraverseFilter[Option, A] _),
       "traverseEither consistent with traverseFilter" -> forAll(
         laws.traverseEitherConsistentWithTraverseFilter[Option, F[A], A, B] _
-      )
+      ),
+      "traverseFilter ref traverseCollect" -> forAll(laws.traverseCollectRef[Option, A, B] _)
```

Comment:
```suggestion
      "traverseFilter reference" -> forAll(laws.traverseCollectRef[Option, A, B] _)
```

https://github.com/typelevel/cats/blob/83ad02b525946c7eb6f74205f6c1968a08a6e75b/laws/src/main/scala/cats/laws/discipline/FoldableTests.scala#L60-L67